src/App.java

```diff
@@ -1,34 +1,264 @@
+        // idea gotten from talking to Gabu
+        // border from https://www.w3schools.com/charsets/ref_utf_symbols.asp and manually formatted
+
+        System.out.println(b+"▌ ▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔▔ ▐\n" +
+                             "▌  ░██╗░░░░░░░██╗███████╗██╗░░░░░░█████╗░░█████╗░███╗░░░███╗███████╗██╗  ▐\n" +
+                             "▌  ░██║░░██╗░░██║██╔════╝██║░░░░░██╔══██╗██╔══██╗████╗░████║██╔════╝██║  ▐\n" +
+                             "▌  ░╚██╗████╗██╔╝█████╗░░██║░░░░░██║░░╚═╝██║░░██║██╔████╔██║█████╗░░██║  ▐\n" +
+                             "▌  ░░████╔═████║░██╔══╝░░██║░░░░░██║░░██╗██║░░██║██║╚██╔╝██║██╔══╝░░╚═╝  ▐\n" +
+                             "▌  ░░╚██╔╝░╚██╔╝░███████╗███████╗╚█████╔╝╚█████╔╝██║░╚═╝░██║███████╗██╗  ▐\n" +
+                             "▌  ░░░╚═╝░░░╚═╝░░╚══════╝╚══════╝░╚════╝░░╚════╝░╚═╝░░░░░╚═╝╚══════╝╚═╝  ▐\n" +
+                             "▌ ▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁ ▐");
+        System.out.print(g + "What is your name?\n"+w+" ↳ ");
+        String userName = madlibs.nextLine();
+        return (userName);
+    }
+
+    // prints out the initial instructions for using the madlibs
+    public void printInstructions() { 
+        System.out.println("| Welcome to your own personal madlibs creation.");
+        System.out.println("| In order for us to create your story, you have to answer a few simple questions.");
+        System.out.println("| These are not personal questions, and feel free to be as creative as possible, so long as the needs we ask for are met.");
+        System.out.println("| A question might prompt you to give a 'Noun' or 'Verb ending in ing', in which case you would type in your answer and press 'ENTER'.");
+        System.out.println("| Please do not include any extra spaces or characters.");
+        System.out.println("| Have fun, and feel free to repeat as many times as you want!");
+        System.out.println(r + "==================");
+    }
+
+    // prints out personalized greeting with userName as a parameter
+    public void printGreeting(String userName) {
+        System.out.println(r + "=================");
+        System.out.println(g + "Hello " + userName + "!");
+    }
     
-    public void printInstructions() {
+    // doMadLib() is responsible for fetching all user inputs in relation to the madlib and for printing out the complete madlib
+    public void doMadlib(String userName){
+
+        System.out.print(g + "Which story would you like to follow?:\n" + b + "[1] --- [2] --- [3]\n" + w);
+        int choice = 0;
+        boolean validChoice = false;
+
+        // Code somewhat copied from https://stackoverflow.com/questions/12702076/try-catch-with-inputmismatchexception-creates-infinite-loop
+        while (!validChoice) {
+            try {
+                choice = madlibs.nextInt(); 
```

Comment:
for future reference, though this clearly works it's usually easier to simply avoid using `.nextInt()` and use `.nextLine()` instead. You can still compare to the string `"1"` or `"2"` or whatever, or even convert to an integer value after the scan, which avoids some of the other annoyances with .nextInt.